md/groovy-in-bonita.md

```diff
@@ -0,0 +1,347 @@
+# Groovy script language in Bonita
+
+In Bonita, it is a prerequisite to be comfortable with Java when implementing advanced behaviors in your processes, the below article aims at helping with Groovy, the scripting language used by Bonita.
+Groovy has been chosen as the scripting language because it is :
+* Concise, readable and have an expressive syntax, easy to learn for Java developers
+* Seamlessly and transparently integrates and interoperates with Java and any third-party libraries
+
+
+## From Java to Groovy
+
+For a complete description, please have a look at the reference documentation [here](http://groovy-lang.org/differences.html).
+
+### Strings
+
```

Comment:
Is it our role to explain what groovy is and what are the language capabilities?

Reply:
We gave our point of view on that, product owner decide.